README.rst

```diff
@@ -789,6 +789,15 @@ Then, add the following command to your CI:
 
     $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection `git diff --name-only --diff-filter=ACMRTUXB $COMMIT_RANGE`
 
+If filenames contain spaces:
+
+.. code-block:: bash
+
+    $ IFS=$'\n'
+    $ files=($(git diff --name-only --diff-filter=ACMRTUXB $COMMIT_RANGE))
+    $ unset IFS
```

Comment:
we have `COMMIT_RANGE`, could we please rename `files` into `COMMIT_SCA_FILES` and wrap this lines into one line ? that way we will ensure that one is calling all 3 of them when he is copy-pasting commands from readme to his CLI